src/api/venue.ts

```diff
@@ -89,25 +89,39 @@ export const updateIframeUrl = async (iframeUrl: string, venueId?: string) => {
 
 type VenueInputForm = Partial<WithId<AnyVenue>> & {
   bannerImageUrl?: string;
-  mapBackgroundImage_url?: string;
-  mapBackgroundImage_file?: FileList;
+  bannerImageFile?: FileList;
+  logoImageUrl?: string;
+  logoImageFile?: FileList;
   numberOfSections?: number;
 };
 
 export const updateVenueNG = async (
   venue: VenueInputForm,
   user: firebase.UserInfo
 ) => {
-  const file = venue.mapBackgroundImage_file?.[0];
-  if (file) {
+  const bannerFile = venue.bannerImageFile?.[0];
+  const logoFile = venue.logoImageFile?.[0];
+
+  if (bannerFile) {
```

Comment:
not sure how to optimize this and similar logic at 116

Reply:
If it works, it's good enough. There are other more generalized image upload takes (if you search for uses of `imageKeys`), but it still boils down to the same procedure.

Reply:
Should check with @denisdimitrov about the storage routs for banners/icons so that there aren't bugs because of legacy routes.